repo/packages/E/elastic/800/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "packagingVersion": "4.0",
+  "upgradesFrom": [
+    "2.7.0-6.8.1"
+  ],
+  "downgradesTo": [
+    "2.7.0-6.8.1"
+  ],
+  "minDcosReleaseVersion": "1.11",
+  "name": "elastic",
+  "version": "2.8.0-7.3.2",
```

Comment:
Also, since this is a major version base-tech upgrade, I just want to double check - there are no breaking changes for customers? They don't have to do anything different for their Elastic 6 service to get to 7 other than 👇  ?
```
dcos elastic update start --package-version=2.8.0-7.3.2
```

Reply:
Update from `2.6.0-6.6.1` -> `2.8.0-7.3.2` is not possible.
```
ph@ph-VirtualBox:~/projects/soak-cluster-configurations$ dcos elastic update start --package-version=2.8.0-7.3.2
dcos-elastic: error: Unable to update elastic to requested version: 2.8.0-7.3.2
Valid package versions are:
- 2.7.0-6.8.1
- 2.5.0-6.3.2, try --help
```
Update from `2.7.0-6.8.1` -> `2.8.0-7.3.2` is ok.

Reply:
Yep, we shouldn't allow from 6.6 to 7, Elastic recommends upgrading to 6.8 before upgrading to 7: https://www.elastic.co/guide/en/elasticsearch/reference/current/setup-upgrade.html

Reply:
hey @phrynchanka - we have control over the upgrade path for our packages. The reason you got an error is bc the perm stub's `marathon.json` specifies:
```
      "upgradesFrom": [
        "2.7.0-6.8.1"
      ],
      "downgradesTo": [
        "2.7.0-6.8.1"
      ],
```
but we can add `6.6.1` as part of the upgrades to/from field. 

That said, Murilo showed that recommended upgrade path from Elastic docs is to get to 6.8.1 first before upgrading to 7 so we can leave this as is.

Reply:
Yep, thanks a lot!